### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SashaViun**
[DEMO LINK](https://SashaViun.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/UserInfo/UserInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  );
+};
```
Comment (NVyrsta) [inline src/components/UserInfo/UserInfo.tsx:13]: destructuring 🙏

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+      >
+        {user && (
+          <UserInfo user={user} />
+        )}
+        ;
```
Comment (NVyrsta) [inline src/components/TodoInfo/TodoInfo.tsx:35]: you print ';' on the page

**Review by NVyrsta — CHANGES_REQUESTED** [review]
Nice work
But try to fix the problem with tests
Also, your styles are different from the sample
You add import App.scss but don't add file App.scss to the project

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Check Demo link , I just see white screen :(

**Pull request closed without merging**